test/test_structured_2d.jl

```diff
@@ -257,6 +257,27 @@ isdir(outdir) && rm(outdir, recursive=true)
       tspan = (0.0, 0.25))
```

Comment:
How long do all the new tests run?

Reply:
Comparing with the latest CI timings from the latest run on `main` and this PR we have

1. tree_mesh_part3: ≈63 min on `main` and ≈65 min on this PR
2. structured_mesh: ≈74 min on `main` and ≈80 min on this PR
3. unstructured_mesh: ≈81 min on `main` and ≈92 min on this PR

I am not sure why adding a single elixir to the unstructured_mesh testing increased the time by so much.

Reply:
That should still be okay